cw-relayer/relayer/messages.go

```diff
@@ -1,173 +1,60 @@
 package relayer
 
 import (
-	"encoding/json"
+	"math/big"
 
-	wasmtypes "github.com/CosmWasm/wasmd/x/wasm/types"
 	"github.com/cosmos/cosmos-sdk/types"
 
 	"github.com/ojo-network/cw-relayer/relayer/client"
+	"github.com/ojo-network/cw-relayer/tools"
 )
 
-type MsgType int
-
-const (
-	RelayRate MsgType = iota + 1
-	RelayHistoricalMedian
-	RelayHistoricalDeviation
-	QueryRateMsg
-	QueryMedianRateMsg
-	QueryDeviationRateMsg
-)
-
-func (m MsgType) String() string {
-	return [...]string{"relay", "relay_historical_median", "relay_historical_deviation"}[m-1]
-}
-
-type (
-	MsgRelay struct {
-		Relay Msg `json:"relay"`
-	}
-
-	MsgForceRelay struct {
-		Relay Msg `json:"force_relay"`
-	}
-
-	MsgRelayHistoricalMedian struct {
-		Relay Msg `json:"relay_historical_median"`
-	}
-
-	MsgForceRelayHistoricalMedian struct {
-		Relay Msg `json:"force_relay_historical_median"`
+func (r *Relayer) genRateMsgs(requestID uint64, resolveTime uint64) (msg []client.PriceFeedData) {
```

Comment:
add a godoc here please - `genRateMsgs is a function which converts x to y for consumption by z`